test/test_matmul_cuda.py

```diff
@@ -572,8 +571,8 @@ def test_float8_rowwise_scaling_sanity(self, device, use_fast_accum: bool) -> No
         x_scales = torch.ones((x.shape[0], 1), device=device, dtype=torch.float32)
         y_scales = torch.ones((1, y.shape[0]), device=device, dtype=torch.float32)
 
-        x_fp8 = x.to(torch.float8_e4m3fn)
-        y_fp8 = y.to(torch.float8_e4m3fn).t()
+        x_fp8 = x.to(e4m3_type)
```

Comment:
I saw that the code to automatically set this was added in https://github.com/pytorch/pytorch/pull/117822 - IMO in a separate PR we should change these dtypes to be set explicitly by the user / testing framework, to follow the convention used elsewhere in similar files and make it crystal clear which dtypes are being tested where.

Reply:
I disagree but am open to being convinced otherwise.  For F8 types there are effectively two sets (e4m3fnuz/e5m2fnuz and e4m3fn/e5m2) and the abstraction introduced in #117822 is useful to avoid much copy/paste code, or decorating many unit tests with allowed types.  I like the current solution because it is compact.  But developer education was missing; new unit tests that work on CUDA and do not use the abstracted types will not work on ROCm.